Jropp/jason-ropp-js-iobp
# Onboarding Project: Webpack.config

## What It Does

This separates and dynamically calls varying configs based on the mode passed to it.

Because new stuffs for projects.

I added a function in webpackconfig that takes the env and dynamically requires and merges (using webpackMerge) production or development config items when needed. 

## How To Test

Build in development mode `$ yarn start`  and then in production mode `$ yarn build`. Notice the difference in file sizes. This is because production mode has a minify plugin applied to it.

Development config holds the server information and also applies a source map.

You can also check out the visual analyze tool with `$ yarn analyze`

## Notes/Caveats

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:


## Dependencies
